ipatests/test_integration/test_ipa_ipa_migration.py

```diff
@@ -260,7 +240,46 @@ def prepare_ipa_server(master):
 
     # Modify passkeyconfig
     master.run_command(
-        ["ipa", "passkeyconfig-mod", "--require-user-verification=FALSE"]
+        [
+            "ipa", "passkeyconfig-mod",
+            "--require-user-verification=FALSE"
+        ]
+    )
```

Comment:
This setting is not used. Did you intend to add a test checking passkeyconfig on the new server?

Reply:
@flo-renaud added a test there.